src/apps/reservation-list/modal/physical-list-details.tsx

```diff
@@ -0,0 +1,242 @@
+import React, {
+  FC,
+  useEffect,
+  useState,
+  useCallback,
+  ChangeEvent
+} from "react";
+import dayjs from "dayjs";
+import EbookIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/Ebook.svg";
+import LocationIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/Location.svg";
+import LoanHistoryIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/LoanHistory.svg";
+import ReservationsIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/Reservations.svg";
+import LoansIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/Loans.svg";
+import { useText } from "../../../core/utils/text";
+import { ReservationType } from "../../../core/utils/types/reservation-type";
+import { MaterialProps } from "../../loan-list/materials/utils/material-fetch-hoc";
+import { useUpdateReservations } from "../../../core/fbs/fbs";
+import {
+  getPreferredBranch,
+  hardcodedInterestPeriods
+} from "../../../components/reservation/helper";
+import { AgencyBranch } from "../../../core/fbs/model";
+import { formatDate } from "../../loan-list/utils/helpers";
+import ListDetails from "../../../components/list-details/list-details";
+import { Button } from "../../../components/Buttons/Button";
+import ListDetailsDropdown, {
+  OptionsProps
+} from "../../../components/list-details-dropdown/list-details-dropdown";
+
+interface PhysicalListDetailsProps {
+  reservation: ReservationType;
+  branches: AgencyBranch[];
+}
+
+const PhysicalListDetails: FC<PhysicalListDetailsProps & MaterialProps> = ({
+  reservation,
+  branches
+}) => {
+  const t = useText();
+  const { mutate } = useUpdateReservations();
+  const [pickupBranchFetched, setPickupBranchFetched] = useState<string>("");
+  const [showBranchesSelect, setShowBranchesSelect] = useState<boolean>(false);
+  const [showExpirySelect, setShowExpirySelect] = useState<boolean>(false);
+  const [newBranch, setNewBranch] = useState<OptionsProps | null>(null);
+  const [newExpiryDate, setNewExpiryDate] = useState<OptionsProps | null>(null);
+  const [branchesOptions, setBranchesOptions] = useState<OptionsProps[] | null>(
+    null
+  );
+  const formatInterestPeriods = Object.entries(hardcodedInterestPeriods(t)).map(
+    ([key, value]) => ({
+      value: key,
+      label: value
+    })
+  );
+
+  const {
+    numberInQueue,
+    pickupBranch,
+    expiryDate,
+    pickupDeadline,
+    dateOfReservation,
+    pickupNumber,
+    reservationId
+  } = reservation;
+
+  useEffect(() => {
+    if (branches && pickupBranch) {
+      // Map branches to match select options
```

Comment:
Nit-picky but we have a commenting standard that says:
// [CAPITAL__FIRST_LETTER][REST_OF_DESCRIPTION_IN_LOWER_CASE].
(notice the period in the end)

Reply:
ha ill fix